pkg/cli/verifier/envoy_config.go

```diff
@@ -461,7 +460,7 @@ func (v *EnvoyConfigVerifier) verifyDestination() Result {
 	return result
 }
 
-func (v *EnvoyConfigVerifier) getDstMeshServicesForSvcPod(svc corev1.Service, podName string) ([]service.MeshService, error) {
+func (v *EnvoyConfigVerifier) getDstMeshServicesForSvcPod(svc corev1.Service) ([]service.MeshService, error) {
```

Comment:
nit: rename function to `getDstMeshServicesForK8sSvc`

Reply:
done